index.html

```diff
@@ -817,14 +823,16 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
 timing buffer current size</a>, set <a>resource timing buffer full
 flag</a> to false.</li>
 </ol>
+
 <p>The attribute <dfn>onresourcetimingbufferfull</dfn> is the event
 handler for the <dfn>resourcetimingbufferfull</dfn> event described
 below.</p>
-<p>To <dfn>add a PerformanceResourceTiming entry</dfn> (<i>new
-entry</i>) in the <a data-cite=
-'PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer'>performance
+
+<p>To <dfn>add a PerformanceResourceTiming entry</dfn> into the <a data-cite=
+'PERFORMANCE-TIMELINE-2/#dfn-performance-entry-buffer'>performance
 entry buffer</a>, run the following steps:</p>
 <ol>
+<li>Let <i>new entry</i> be the input <a>PerformanceEntry</a> to be added.
 <li>If <a>resource timing buffer current size</a> is less than
```

Comment:
Hmm I think we also need to check that the secondary buffer is empty here? Otherwise we're adding entries in the incorrect order. That is, if we get full, queue a task, and add some entries into secondary buffer but then clear the primary buffer and then get a new entry, we'll add that new entry to the primary buffer before the ones in the secondary buffer.

Reply:
Yeah, we need to check the `resource timing buffer full flag` before adding to the primary buffer